test/tests_for_integration/api_to_db_test.py

```diff
@@ -607,3 +617,149 @@ def _run_job_fail(ee2_port, token, params, expected, throw_exception=False):
         with raises(ServerError) as got:
             client.run_job(params)
         assert_exception_correct(got.value, ServerError("name", 1, expected))
+
+
+######## run_job_concierge tests ########
+
+
+def test_run_job_concierge_minimal(ee2_port, ws_controller, mongo_client):
+    def modify_sub(sub):
+        del sub["Concurrency_Limits"]
+
+    _run_job_concierge(
+        ee2_port,
+        ws_controller,
+        mongo_client,
+        # if the concierge dict is empty, regular old run_job gets run
+        {"trigger": "concierge"},  # contents are ignored
+        modify_sub,
+        "concierge",
+        4,
+        23000,
+        100,
+    )
+
+
+def test_run_job_concierge_mixed(ee2_port, ws_controller, mongo_client):
+    """
+    Gets cpu from the input, memory from deploy.cfg, and disk from the catalog.
+    """
+
+    def modify_sub(sub):
+        del sub["Concurrency_Limits"]
+
+    _run_job_concierge(
+        ee2_port,
+        ws_controller,
+        mongo_client,
+        {"client_group": "extreme", "request_cpus": 76},
+        modify_sub,
+        "extreme",
+        76,
```

Comment:
I think it would improve readability to pass in the variable names for this longer function signatures, like conc_params=modify_sub, cpu=76, memory=25000, disk=11

Reply:
done